app/jobs/imo/invite_patient.rb

```diff
@@ -7,6 +7,12 @@ def perform(patient_id)
     return unless Flipper.enabled?(:imo_messaging)
 
     patient = Patient.find(patient_id)
-    ImoApiService.new.send_invitation(patient)
+    status = ImoApiService.new.send_invitation(patient)
```

Comment:
Should we check the status here?  If Imo returns some non 200 result code (and we get back and `:error` for status), I don't think we'd want to create or update the ImoAuthorization below.